app/scripts/templates/sign_up_password.mustache

```diff
@@ -21,7 +21,7 @@
 
       <div class="input-row password-row">
         <input id="password" type="password" class="password check-password tooltip-below" placeholder="{{#t}}Password{{/t}}" value="{{ password }}" pattern=".{8,}" required autofocus data-form-prefill="password" data-synchronize-show="true" />
-        <div class="input-help input-help-focused input-help-signup">{{#t}}A strong, unique password will keep your Firefox data safe from intruders.{{/t}}</div>
+        <div class="input-help input-help-focused input-help-signup">{{#t}}Tips: Keep your password safe, and don’t reuse your email password.{{/t}}</div>
```

Comment:
Found this: https://github.com/mozilla/fxa-content-server/issues/5947 

will need a follow up PR with more stuff to add that in